doc/CONTRIBUTING.md

```diff
@@ -102,6 +102,20 @@ Good commit message subject :
 Document how to install the project
 ```
 
+Also, when you specify what you did avoid commit message subjects with "Fix bug
+in such and such feature". Saying you are fixing something implies the previous
+implementation was wrong and yours is right, which might not be even true.
+Instead, humbly say what you did technically and **then** explain how this is
```

Comment:
I'm not sure this is worth saying.
When you propose a change, you want to fix a bug and you do this because you encountered the bug, so the previous implementation was indeed wrong in some way (or you think it was).
Or you refactor things because an implementation is wrong/incomplete/bloated/... (or you think it is).
Saying that you think you're right does not imply you're not humble, you're just stating your point.

IMO this is not the reason why writing commit messages this way is better.